ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -1185,7 +1185,7 @@ export default defineComponent({
           ? await getMHRegistrationSummary(getMhrInformation.value.mhrNumber, false)
           : null
 
-        if (!!regSum && !!regSum.lienRegistrationType) {
+          if (!!regSum && !!regSum.lienRegistrationType && !isRoleStaffReg.value) {
```

Comment:
Looks good but looks like indent/spacing is slightly off

Reply:
Thanks @cameron-eyds , I fixed removed the indent.
I am gonna merge the PR now.